src/test/a11y/tests/admin-routes.test.ts

```diff
@@ -83,4 +83,180 @@ describe('Accessibility - Admin Routes', () => {
+            });
+
+            describe('with invalid input data', () => {
+                testAccessibility(url, '', true, { lastName: 'Invalid lastName' });
+            });
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { firstName: '' });
+            });
+
+            describe('with invalid input data', () => {
+                testAccessibility(url, '', true, { firstName: 'Invalid firstName' });
+            });
+        });
+    });
+
+    describe('Admin Management Page', () => {
+        const url = '/admin-management';
+
+        describe('with no input data', () => {
+            testAccessibility(url, '', true, { 'search-input': '' });
+        });
+
+        describe('with invalid input data', () => {
+            testAccessibility(url, '', true, { 'search-input': 'Not found' });
+        });
+    });
+
+    describe('Create Admin Account Summary Page', () => {
+        const url = '/create-admin-account-summary';
+
+        describe('with no input data', () => {
+            testAccessibility(url, '', true, { 'view-choice': '' });
+        });
+    });
+
+    describe('Update User Page', () => {
+        const url = '/update-user';
+
+        describe('with no input data', () => {
+            testAccessibility(url, '', true, { errorMessage: '' });
+        });
+
+        describe('with invalid input data', () => {
+            testAccessibility(url, '', true, { errorMessage: 'Incorrect Title' });
+        });
+    });
+
+    describe('Manual Upload Page', () => {
+        const url = '/manual-upload';
+
+        describe('with no input data', () => {
+            testAccessibility(url, '', true, { 'manual-file-upload': '' });
+        });
+
+        describe('with invalid input data', () => {
+            testAccessibility(url, '', true, { fileErrorMessage: 'Incorrect file format' });
+        });
+
+        describe('with invalid input data', () => {
+            testAccessibility(url, '', true, { courtError: 'Incorrect Court' });
+        });
+
+        describe('with invalid input data', () => {
+            testAccessibility(url, '', true, { contentDateError: 'Incorrect Content Date' });
+        });
+
+        describe('with invalid input data', () => {
+            testAccessibility(url, '', true, { classificationError: 'Incorrect Classification' });
+        });
+
+        describe('with invalid input data', () => {
+            testAccessibility(url, '', true, { displayDateError: 'Incorrect Display Date' });
+        });
+
+        describe('with invalid input data', () => {
+            testAccessibility(url, '', true, { displayDateFrom: 'Incorrect From Date' });
+        });
+
+        describe('with invalid input data', () => {
+            testAccessibility(url, '', true, { displayDateTo: 'Incorrect To Date' });
+        });
+    });
+
+    describe('Manual Upload Summary Page', () => {
+        describe('A-Z Search Page', () => {
```

Comment:
why is there is 'describe' for A-Z search page but u r testing manual-upload-summary